prevayler4/git-coherence/project.clj

```diff
@@ -0,0 +1,15 @@
+(defproject house.jux/prevayler4.git-coherence "0.0.1-SNAPSHOT"
```

Comment:
when should we change the snapshot version?

Reply:
We do a commit with a single line change, changing the version to a date like 2025.02.12, when we believe the lib is ready to be used.